buildSrc/src/main/kotlin/convention.publishing.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import com.redmadrobot.build.dsl.*
+
+plugins {
+    id("com.redmadrobot.publish-config")
+    id("com.redmadrobot.publish")
+}
+
+group = "com.redmadrobot.extensions"
+version = "1.0.0"
```

Comment:
It might be better to keep version and group in a more "observable" place. For example, we can configure it in root project and then copy to children using `version = rootProject.version`, or event more Gradle-native solution is to place group and version into `gradle.properties` file as it [done here](https://github.com/RedMadRobot/gradle-version-catalogs/blob/main/gradle.properties#L1-L3)